app/src/main/java/org/greenstand/android/TreeTracker/orgpicker/OrgPickerScreen.kt

```diff
@@ -34,6 +35,7 @@ import org.greenstand.android.TreeTracker.view.TreeTrackerButton
 
 @OptIn(ExperimentalFoundationApi::class)
 @Composable
+@Preview(showBackground = true)
```

Comment:
This preview doesn't work. You can either get it working, or remove this annotation.